src/components/Buttons/ButtonTag.tsx

```diff
@@ -11,35 +11,31 @@ type ButtonTagProps = {
   dataCy?: string;
 };
 
-const ButtonTag: React.FunctionComponent<ButtonTagProps> = ({
-  onClick,
-  selected,
-  children,
-  size,
-  removable = false,
-  dataCy
-}) => {
-  const className = clsx(
-    "tag",
-    selected && "tag--fill",
-    size && `tag--${size}`,
-    "cursor-pointer"
-  );
+const ButtonTag = React.forwardRef<HTMLButtonElement, ButtonTagProps>(
+  ({ onClick, selected, children, size, removable = false, dataCy }, ref) => {
```

Comment:
Is there a reason why we don't destructure the ref inside the curly brackets? 🤔

Reply:
As I understand it, `ref` is not a prop in the typical sense, but a way to interact directly with a DOM element, as described here: https://react.dev/reference/react/forwardRef

Maybe something like this would be more clear? 🙂

```

const ButtonTag = React.forwardRef<HTMLButtonElement, ButtonTagProps>(
  (props, ref) => {
    const {
      onClick,
      selected,
      children,
      size,
      removable = false,
      dataCy
    } = props;
```